data/crac-file/crac-api/src/main/java/com/farao_community/farao/data/crac_api/RangeAction.java

```diff
@@ -18,4 +20,7 @@
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.MINIMAL_CLASS)
 public interface RangeAction extends RemedialAction, ApplicableRangeAction, Range {
+    List<ApplicableRangeAction> getApplicableRangeActions();
+
+    List<NetworkElement> getNetworkElements();
```

Comment:
So you can remove these methods here. RangeAction and NetworkAction are just the generic interfaces gathering the others to gather methods so it seems that they won't have their own methods.